examples/js/exporters/GLTFExporter.js

```diff
@@ -964,12 +985,30 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
-			// pbrMetallicRoughness.baseColorTexture
+			// pbrMetallicRoughness.baseColorTexture or pbrSpecularGlossiness diffuseTexture
 			if ( material.map ) {
 
 				var baseColorMapDef = { index: processTexture( material.map ) };
 				applyTextureTransform( baseColorMapDef, material.map );
-				gltfMaterial.pbrMetallicRoughness.baseColorTexture = baseColorMapDef;
+
+				if ( material.isGLTFSpecularGlossinessMaterial ) {
+
+					gltfMaterial.extensions.KHR_materials_pbrSpecularGlossiness.diffuseTexture = baseColorMapDef;
+
+				} else {
+
+					gltfMaterial.pbrMetallicRoughness.baseColorTexture = baseColorMapDef;
+
+				}
+				
+			}
+
+			// pbrSpecularGlossiness specular map
+			if ( material.specularMap ) {
```

Comment:
Some other threejs material types actually do have a `specularMap` property, like MeshLambertMaterial; we shouldn't enter this if statement unless it's also definitely a pbrSpecularGlossiness material.